util/Directories.cpp

```diff
@@ -635,3 +635,55 @@ fs::path GetPath(const std::string& path_string) {
     }
     return GetPath(path_type);
 }
+
+bool IsExistingFile(const fs::path& path) {
```

Comment:
what does this do differently from just calling `fs::is_regular_file(path)` ?

Reply:
The main use of such functions is work towards consolidating filesystem operations.
The are declaration compatible excerpts from larger work towards VFS.

Without considering the above (as it has not been discussed in depth), for this one function:
* Eliminates need to include boost/filesystem/operations.hpp at call site
* Short-circuits if `fs:exists` fails (with single stat call)
* Ensures filesystem errors are handled